policyengine_us/parameters/gov/states/az/tax/income/credits/increased_excise_tax_credit/dependent2.yaml

```diff
@@ -0,0 +1,29 @@
+description: Arizona accounts for the following number of dependents for the increase excise tax credit based on the filing status.
+metadata:
+  period: year
+  unit: /1
+  label: Arizona Increase excise tax credit calculated dependents 
+  breakdown:
+    - filing_status
+  reference:
+    - title: Taxation of Income § 43-1072.01 - Credit for increased excise taxes paid
+      href: https://www.azleg.gov/viewdocument/?docName=https://www.azleg.gov/ars/43/01072-01.htm
+    - title: AZ tax-140 From Year 2022 Line 56 Page 25
+      href: https://azdor.gov/forms/individual/form-140-arizona-resident-personal-income-tax-booklet 
+    - title: AZ tax-140 From Year 2021 Line 56 Page 24
+      href: https://azdor.gov/forms/individual/form-140-arizona-resident-personal-income-tax-booklet
+SINGLE:
+  2021-01-01: 1
+  2022-01-01: 1
+HEAD_OF_HOUSEHOLD:
+  2021-01-01: 2
+  2022-01-01: 2
```

Comment:
```suggestion
```

Reply:
if the value remains the same we only use the older - 2021 - value as it is set to this from that moment on